crates/sui-indexer/src/schema_v2.rs

```diff
@@ -341,8 +325,4 @@ diesel::allow_tables_to_appear_in_same_query!(
     tx_input_objects,
     tx_recipients,
     tx_senders,
-    tx_indices,
 );
-
-use diesel::sql_types::Text;
```

Comment:
@oxade is `query_cost` necessary here? This PR removed it but passed all CI tests.

Reply:
We disabled it in code so no test triggers it.
But yeah I wouldn't remove it just yet. Let minimize potential DB conflicts.

Reply:
We don't use the custom sql fn query_cost anymore, so we can consider removing it from our test code instead

Reply:
I am not quite following here, but feel free to add it back if it is necessary